cmd/registry/cmd/root.go

```diff
@@ -53,5 +67,6 @@ func Command(ctx context.Context) *cobra.Command {
 	cmd.AddCommand(upload.Command(ctx))
 	cmd.AddCommand(vocabulary.Command(ctx))
 
+	cmd.PersistentFlags().StringVar(&taskID, "task_id", "", "Assign an ID to this execution")
```

Comment:
Can we name this something more generic like `log_id` and mention in the description that it gets attached to the program's logs? Right now it seems unclear what assigning an ID actually does and may confuse users.

Reply:
Done